### ISSUE TITLE
Person Verifier Showcase

### ISSUE THREAD

**Issue by cvarjao** [bcgov/BC-Wallet-Demo#85]

**Comment by wadeking98** [comment]
Deployed at https://bc-wallet-demo-dev.apps.silver.devops.gov.bc.ca/digital-trust/showcase/person
assigning to Alex and Kim for UX work

**Comment by cvarjao** [comment]
@wadeking98, while we are waiting for UX work, please remove "picture" and "Expiry Date Dateint" attributes form the proof request

**Comment by wadeking98** [comment]
Are we still using this now that we have the lawyer2 demo?

**Comment by hannahgov** [comment]
@cvarjao Is this Showcase different from lawyer 2?

**Comment by cvarjao** [comment]
@hannahgov, yes. This is a verifier only. The lawyer 2 also issues demo credentials. But, this is till an idea phase, the new lawyer 2 may satisfy

**Comment by jleach** [comment]
Stale. Closing.
